.github/workflows/release.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     runs-on: macos-13
     steps:
       - uses: actions/checkout@v4
+      - name: Select Xcode version
+        # https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#xcode
+        run: sudo xcode-select --switch /Applications/Xcode_15.0.app
```

Comment:
@rebello95 Any reason we wouldn't want to use `Xcode_15.0.1`?

Reply:
We can, but was just updating to match the rest of the CI jobs for now which use 15.0

Reply:
(This one job was using the default on the GitHub runner, whereas the rest specify a version)